packages/admin/cms-admin/src/dam/DataGrid/thumbnail/DamThumbnail.tsx

```diff
@@ -23,6 +24,15 @@ const ImageThumbnail = styled("img")`
     border-radius: inherit;
 `;
 
+const ImagePreview = styled("img")`
+    max-width: 300px;
+    max-height: 300px;
+    padding: 0.4rem;
+    margin-left: 0.25rem;
```

Comment:
We usually don't use `rem` for padding, but `px`. And could you use `theme.padding()` for this?